wiki/osu!stream/en.md

```diff
@@ -1,14 +1,22 @@
 # osu!stream
 
+*For the list of songs featured in the game, see: [osu!stream song list](/wiki/osu!stream/Song_List)*
+
 ![](img/Os-Logo.jpg "osu!stream logo")
 
-**osu!stream** is a version of osu! built from scratch that is made especially for touch-enabled devices. It is currently available for Apple's [iOS](http://en.wikipedia.org/wiki/IOS) platform only. An [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) port was planned and nearly completed, but has yet to be released due to audio latency issues in Android causing an unenjoyable gameplay experience.
+**osu!stream** is a special version of osu! made for handheld [iOS](http://en.wikipedia.org/wiki/IOS) and [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) devices. It is available for download from the [App Store](https://apps.apple.com/us/app/osu-stream/id436952197) (iOS) and [Play Store](https://play.google.com/store/apps/details?id=sh.ppy.osustream&pli=1) (Android) free of charge.
+
+Unlike regular osu!, osu!stream features a completely new gameplay system, with new gameplay elements such as hold circles and connected beats. It does not support classic osu! beatmaps without manual conversion. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a [stricter level of standards](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
```

Comment:
also, `are created` -> `were created` to emphasize they are no longer being created?
```suggestion
Unlike regular osu!, osu!stream features a completely new gameplay system with unique elements such as hold circles and connected beats. It does not support classic osu! beatmaps without manual conversion. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a [stricter level of standards](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
```